WalletWasabi.Fluent/Views/MainView.axaml

```diff
@@ -34,7 +34,7 @@
       </SplitView.Pane>
       <Panel>
         <DockPanel>
-          <local:StatusBar DockPanel.Dock="Bottom" DataContext="{Binding StatusBar}" VerticalAlignment="Bottom" Height="32" TextBlock.FontSize="12" />
+          <local:StatusBar Classes="dpbottom vbottom h7" DataContext="{Binding StatusBar}" Height="32" />
```

Comment:
I dont like using the dockpanel and alignment for things like this... styles use memory and are not performant.. here explicitly setting these things make sense...

lets make this a text only thing?

Reply:
@danwalmsley i'd like to retain the v/h alignment helper styles for text and etc. if it's alright